modules/drivers/athena/src/metabase/driver/athena.clj

```diff
@@ -315,58 +315,50 @@
     (catch Exception e
       (log/error (u/format-color 'red "Failed to execute query: %s %s" query (.getMessage e))))))
 
-(defn- describe-database->clj
-  "Workaround for wrong getColumnCount response by the driver (huh?)"
-  [column-metadata]
-  {:name (str/trim (:col_name column-metadata))
-   :type (str/trim (:data_type column-metadata))})
```

Comment:
we can avoid nonsense like this by just fetching the columns as the keys we want in the first place